bench_runner/templates/benchmark.src.yml

```
@@ -127,7 +131,7 @@ jobs:
 
   publish:
     uses: ./.github/workflows/_publish.yml
-    if: ${{ always() }}
+    if: ${{ inputs.publish }}
```

The `always()` is there so it always publishes even if one of the prior benchmarking jobs fails to complete.  That's really useful so that one flaky machine doesn't bring everyone else down.

I wonder (but am not sure), if this would work:

```suggestion
    if: ${{ inputs.publish && always() }}
```

---

Same as above, this change is no longer needed.